test/test_respondent_view.py

```diff
@@ -0,0 +1,31 @@
+import json
+from test.party_client import PartyTestClient
+from test.test_data.default_test_values import (
+    DEFAULT_BUSINESS_UUID,
+    DEFAULT_SURVEY_UUID,
+)
+from test.test_data.mock_respondent import MockRespondent
+from unittest.mock import patch
+
+
+class TestRespondentView(PartyTestClient):
```

Comment:
As above, I have made a class just for the view, realistically we should separate the tests out, but considering the amount it might just be doing it from now on, and not extending a bad pattern

Reply:
This looks fine to me and agree with separating this out from the controllers. The only issue I have with doing this is when the views are complex and become too dependent on vast amounts of drifted mocked json. However this feels appropriate.